theme/home.less

```diff
@@ -54,26 +54,53 @@
         }
     }
     .getting-started-segment.hero {
-        .action {
+        -webkit-transition: background-image 0.2s ease-in;
+        transition: background-image 0.2s ease-in;
+        overflow: hidden;
+
+        .hero-banner-contents {
             position: absolute;
-            left: 1rem;
-            bottom: 1rem;
-            margin: 0.5rem;
+            top: 0;
+            left: 0;
+            height: 100%;
+            width: 20rem;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            margin-left: @carouselArrowSize;
         }
-        .dots {
-            position: absolute;
-            bottom: 2rem;
-            right: 2rem;
 
-            button {
-                margin-right: 1.5rem;
-            }
+        .description {
+            width: 20rem;
+            font-size: 1.5rem;
+            font-weight: 600;
+            color: @white;
+        }
+
+        .action {
+            margin: 1rem 0;
+        }
+
+        .dots button {
+            border: 2px solid #6D6D6D;
+            margin-right: 1.25rem;
 
-            button.active {
-                background: @primaryColor;
+            &.active {
+                border-color: #aaaaaa;
+                background: none;
             }
         }
+
+        .gradient-overlay {
+            position: relative;
+            height: 100%;
+            width: 40rem;
+            top: 0;
+            left: 0;
+            background: linear-gradient(to right, rgba(0, 0, 0, 1) 0%, rgba(0, 0, 0, 0) 100%);
```

Comment:
small: should the color we're fading to be a less variable/configurable per theme?